ride-service/src/main/kotlin/com/dulfinne/taxi/rideservice/controller/DriverController.kt

```diff
@@ -29,71 +29,71 @@ class DriverController(val service: DriverService) {
         return ResponseEntity.ok(ridesResponsePage)
     }
 
-    // TODO: Get {driverId} from token
-    @PostMapping("/{driverId}/accept/{rideId}")
+    // TODO: Get {driverUsername} from token
+    @PostMapping("/{driverUsername}/accept/{rideId}")
     fun acceptRide(
         @PathVariable("rideId") rideId: Long,
-        @PathVariable("driverId") driverId: Long
+        @PathVariable("driverUsername") driverUsername: String
```

Comment:
What it the reason of such edit?

Reply:
I'm using driverUsername and passengerUsername because they're like unique identifier from Keycloak, and I can get them from token. This edit is already approved in other services(driver and passenger), and my security logic built on this.

Reply:
Ok, but for future - this is a bad solution. 

Better to add attrribute `user-id` in Keycloak and set your id in it. Then it will appear in your access token

Reply:
In this case (`username`) you are driven by Keycloak rules. What if you change the provider and it will use ids?